docs/development_guide/README.md

```diff
@@ -102,19 +144,18 @@ Some of the scripts and build targets available expect an accessible container r
 Additionally, if using `demo/` scripts to deploy OSM's provided demo on your own K8s cluster, the same container registry configured in `.env` will be used to pull OSM images on your K8s cluster.
 
 ```console
-$ # K8S_NAMESPACE is the Namespace the control plane will be installed into
-$ export K8S_NAMESPACE=osm-system
-
-$ # CTR_REGISTRY is the URL of the container registry to use
-$ export CTR_REGISTRY=<your registry>
+# K8S_NAMESPACE is the Namespace the control plane will be installed into
+export K8S_NAMESPACE=osm-system
```

Comment:
the rest of the doc didn't have the `$`'s in the code snippets. imo it's easier to copy these to console if they are there